tests/unit/s2n_seccomp_handshake_test.c

```diff
@@ -19,7 +19,14 @@
 
 int main(int argc, char **argv)
 {
-    BEGIN_TEST_NO_INIT();
+    /* We need to execute BEGIN_TEST() before the seccomp filter is applied.
```

Comment:
The previous phrasing was correct. s2n_init is the part of BEGIN_TEST that we specifically need to occur before the seccomp filter is applied. And if you change it to BEGIN_TEST, the part about an application needing to do the same doesn't make sense.